app/src/common/shared/com/igalia/wolvic/browser/engine/Session.java

```diff
@@ -74,6 +75,7 @@ public class Session implements WContentBlocking.Delegate, WSession.NavigationDe
     private static UriOverride sUserAgentOverride;
     private static UriOverride sDesktopModeOverrides;
     private static final long KEEP_ALIVE_DURATION_MS = 1000; // 1 second.
+    private static final String DEFAULT_USER_AGENT = "Mozilla/5.0 (Android 10; Mobile VR; rv:105.0) Gecko/105.0 Wolvic/" + BuildConfig.VERSION_NAME;
```

Comment:
I'd be awesome to get the Gecko version automatically from build configuration. I don't think there is an easy way to do that ATM I guess. 

Anyway we'd have to change this everytime we update Gecko and we should take that into account